README.md

```diff
@@ -233,6 +235,27 @@ Need to setup Flatpak (https://flatpak.org/setup/):
 * Chrome OS
 
 
+Systemd
+-------
+If your distribuition is running under systemd and you want to generate and upload
+hw-probe report automatically, please install as root:
+
+    cp -a systemd/* $(systemdsystemunitdir)/
+
+Normally systemd units dir is located at `/usr/lib/systemd/system`
+You may want to get systemd unit dir by running
+`pkg-config --variable=systemdsystemunitdir systemd`
+
+Enable hw-probe.timer by running as root:
+
+    systemctl enable --now hw-probe.timer
+
+This timer will execute one time per month a hw-probe.service that will generate and 
+upload report to https://linux-hardware.org
+
+User may edit hw-probe.timer and change OnCalendar value to daily, monthly... to execute
```

Comment:
Let's set minimal period to month. Please don't mention daily and weekly runs. This is too often and unnecessary.

Reply:
It is already set to monthly run. Yes I may remove references to shorter periods plus warning message that lowering this value is stronly not recommended and useless.

Reply:
Done